frontend/src/components/Modal/InviteToCollectionModal/InviteToCollectionModal.js

```diff
@@ -0,0 +1,175 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { Modal, Button, Form } from "react-bootstrap";
+import PropTypes from "prop-types";
+
+import { authActions, userActions, collectionActions } from "../../../store/actions";
+import UserEntry from "../../User/UserEntry/UserEntry";
+
+class InviteToCollectionModal extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            searchKeyWord: "",
+            isModalOpen: false,
+            isSearchResult: false,
+            checkedUserIdList: [],
+        };
+    }
+
+    componentDidMount() {
+        this.props.onGetMe()
```

Comment:
You should not call getMe in any components other than Header. It's redundant and unnecessary. I mentioned it in your last PR.

Reply:
You can refer to AccountSetting for it. Also, CollectionList can be improved in this respect.

Reply:
Thank you for letting me know how to use 'componentDidUpdate'. I improved the code by your suggestion.